src/services/administration/iam/role/update-role/modules/RoleUpdateForm.vue

```diff
@@ -35,14 +47,37 @@ export default {
         PolicyListDataTable,
         PPanelTop,
     },
-    setup() {
+    setup(props, { emit }) {
+        const {
+            forms: {
+                selectedPolicyIdList,
+            },
+            setForm,
+            invalidTexts,
+            isAllValid: isPolicySectionValid,
+        } = useFormValidator({
+            selectedPolicyIdList: [],
+        }, {
+            selectedPolicyIdList(value: string[]) { return value.length ? ' ' : i18n.t('At least one API policy must be attached.'); }, // song-lang
+        });
         const state = reactive({
-            selectedPolicyIdList: [] as string[],
+            isBaseInformationValid: false,
+            isAllValid: computed(() => isPolicySectionValid && state.isBaseInformationValid),
+        });
+        const handleUpdate = (value) => { setForm('selectedPolicyIdList', value); };
+        const handleBaseInfoValidate = (value: boolean) => { state.isBaseInformationValid = value; };
+
+        watch(() => state.isAllValid, (after) => {
+            emit('validated', after);
```

Comment:
```suggestion
            emit('update-validation', after);
```

제가 제안한 이름일 필욘 없지만, 리뷰를 하다보니 헷갈려서 제안드려요.
"이 폼은 유효하다" 라는 이벤트명 보다는, "유효성 검사결과가 바꼈다" 라는 이벤트 명이 적절해보입니다.